app/controllers/users.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const userService = require('../services/users'),
+  errors = require('../errors');
+
+const checkAlphanumeric = (string) => {
+  const regex = /^[a-zA-Z0-9]*$/;
+  return regex.test(string);
+};
+
+const validateEmail = (email) => {
+  const regex = /\S+@\S+\.\S+/;
+  return regex.test(email);
+};
+
+const emailValidation = (email) => {
+  if (!validateEmail(email)) {
+    return 'The requested email is not a valid email';
```

Comment:
Instead of returning the string you could throw an exception here and avoid the check if there is some validation error or not in the signup method.